osf/migrations/0198_add_egap_flag.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2019-07-23 14:26
+from __future__ import unicode_literals
+from django.db import migrations
+
+from osf import features
+from osf.utils.migrations import AddWaffleFlags, DeleteWaffleSwitches
+
+class Migration(migrations.Migration):
```

Comment:
Reminder to rename migration to try to increment the numerical names of migrations on develop

Reply:
as well as update the dependency